go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/EXCCoin/exccdata
 require (
 	github.com/AndreasBriese/bbloom v0.0.0-20180913140656-343706a395b7 // indirect
 	github.com/EXCCoin/base58 v0.0.0-20180515090142-e1a805ee5d9f // indirect
-	github.com/EXCCoin/exccd v0.0.0-20180907111253-e377db154d35
-	github.com/EXCCoin/exccwallet v1.0.0
+	github.com/EXCCoin/exccd v0.0.0-20180725103841-d3143e42e2ad
```

Comment:
Please update exccd dependencies with `go get -u`. Currently, it points to July version.